lib/rules/indent.js

```diff
@@ -850,7 +850,18 @@ module.exports = {
 
             ExportNamedDeclaration(node) {
                 if (node.declaration === null) {
-                    addElementListIndent(getTokensAndComments(node).slice(1), node.specifiers, 1);
+                    const tokensInNode = getTokensAndComments(node);
+                    const closingCurly = sourceCode.getLastToken(node, astUtils.isClosingBraceToken);
+                    const closingCurlyIndex = lodash.sortedIndexBy(tokensInNode, closingCurly, token => token.range[0]);
```

Comment:
Wondering if the performance would be improved a little by using [`lodash.sortedLastIndexBy`](https://lodash.com/docs/#sortedLastIndexBy)? We know the closing curly should be near the end of the token array.